transformers/events/vow_flog/converter.go

```diff
@@ -40,7 +40,7 @@ func (VowFlogConverter) ToModels(_ string, logs []core.HeaderSyncLog) ([]shared.
 		era := log.Log.Topics[2].Big()
 
 		model := shared.InsertionModel{
-			SchemaName: "maker",
+			SchemaName: constants.MakerSchema,
 			TableName:  "vow_flog",
```

Comment:
Also doesn't need to happen in this PR, but could be cool to reuse the labels from the constants for all of the table names. Perhaps even renaming `Label` to `Table`